utilities/bash/run_six.sh

```diff
@@ -930,7 +940,24 @@ function checkDirReadyForSubmission(){
 	    workunitName="${fileNames[0]}"
 	    sixdeskmess  1 ".desc and .zip files present in $RundirFullPath!"
 	fi
-    fi
+    elif [ "$sixdeskplatform" == "htboinc" ] ; then
```

Comment:
The respective lines in case of the BOINC platform are for matching the file names of the .zip and .desc. Since for HTBOINC we generate only the .zip, I would not bother with this check - moreover, the generation of the .zip file is already under quite a bunch of conditional statements